CMakeLists.txt

```diff
@@ -519,7 +526,8 @@ print_config("Valgrind             :" ENABLE_VALGRIND)
 message("")
 print_config("Shared libs          :" ENABLE_SHARED)
 print_config("Static binary        :" ENABLE_STATIC_BINARY)
-print_config("Java bindings        :" BUILD_BINDINGS_JAVA)
+print_config("Java bindings        :" BUILD_SWIG_BINDINGS_JAVA)
+print_config("Python bindings      :" BUILD_SWIG_BINDINGS_PYTHON)
```

Comment:
This seems to print "Python bindings" twice. Maybe we can have two separate titles?

Reply:
Good feedback! What do you think of the fix in 395c653074dad07b42e2e8785f6bc2e205179f86? Basically just calls them "SWIG bindings" if they're generated by SWIG.

Reply:
That works for me :+1: